src/index.js

```diff
@@ -189,6 +189,7 @@ class ReactTooltip extends React.Component {
     let domRoot;
 
     switch (parentNode.constructor.name) {
+      case 'Document':
```

Comment:
Could you explain why this fixes the issue?

Reply:
I would love to know as well

Reply:
Because without this, the code tries to insert child style element at that wrong place therefore leading to exception whereas Document and HTMLDocument both have a header section where the style child can be added without any issue

Reply:
I think the problem is that `document.constructor.name` property returns different results for browser and jsdom environments. 
In browser it is 'HTMLDocument' so injectStyles works perfectly:
![image](https://user-images.githubusercontent.com/5908054/112426299-8a7d3800-8d59-11eb-8fba-b2a48367dc21.png)
and in jsdom it is just 'Document':
![image](https://user-images.githubusercontent.com/5908054/112426493-d9c36880-8d59-11eb-8bc6-8b6f816a8b5f.png)
which leads to default case and sets domRoot to be document
and calling appendChild on 'document' leads to DOMException (HierarchyRequestError: The operation would yield an incorrect node tree.)
This PR fixes this problem